cmd/main.go

```diff
@@ -96,6 +96,23 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "PolicyRecommendation")
 		os.Exit(1)
 	}
+
+	if err = (&controller.PolicyRecommendationRegistrar{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "PolicyRecommendationRegistration")
+		os.Exit(1)
+	}
+
+	if err = (&controller.BreachMonitorTrigger{
```

Comment:
I opine we should have a different controller manager for triggers to decouple the deployment of triggers and reco workflow execution engine. What do you say?

Reply:
IMO it will be just overhead. Is there a specific reason to separate the controller manager if all of them are being deployed in the same binary.